cfgov/v1/migrations/0006_update_tables.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from __future__ import print_function
+from itertools import chain
+
+from django.db import migrations
+
+from v1.models.learn_page import LearnPage, DocumentDetailPage
```

Comment:
Do not import the models directly. Use the get_model method to avoid data migration failure in the future.

```
Model = apps.get_model("yourappname", "Model")
```

https://docs.djangoproject.com/en/1.10/topics/migrations/#data-migrations